e2e/rest-client-e2e/tests/bump-version.spec.ts

```diff
@@ -0,0 +1,56 @@
+import {
+    checkFilesExist,
+    runNxCommand,
+    runNxCommandAsync,
+    uniq,
+} from '@nrwl/nx-plugin/testing';
+import { newProject, cleanup } from '@ensono-stacks/e2e';
+
+describe('http-client e2e', () => {
+    beforeAll(async () => {
+        await newProject(['@ensono-stacks/rest-client'], ['@nrwl/js']);
+    });
+
+    afterAll(() => {
+        cleanup();
+    });
+
+    describe('client-endpoint', () => {
```

Comment:
Creating a new Project is expensive. could we just have these as separate tests in one spec file?

before: create new Project
test1: create client
test2: create endpoint
test3: bump version endpoint

What do you think?

Reply:
I agree actually, it ruins a separation of concern which I like to introduce in my files but it makes sense.